recipe/meta.yaml

```diff
@@ -22,7 +22,7 @@ requirements:
     - poetry-core
     - python >=3.7,<4.0
   run:
-    - copier >=8.0.0,<9.0.0
+    - copier >=8.2.0,<9.0.0
```

Comment:
The issue is that copier 8.2 and above needs pydantic 2 and above.

https://github.com/conda-forge/copier-feedstock/pull/16

Reply:
Thanks! Looks like I missed removing that pin...